internal/services/azurestackhci/stack_hci_deployment_setting_resource.go

```diff
@@ -0,0 +1,1365 @@
+															ForceNew:     true,
+															ValidateFunc: validation.StringIsNotEmpty,
+														},
+
+														"network_direct": {
+															Type:         pluginsdk.TypeString,
+															Optional:     true,
+															ForceNew:     true,
+															ValidateFunc: validation.StringIsNotEmpty,
+														},
+
+														"network_direct_technology": {
+															Type:         pluginsdk.TypeString,
+															Optional:     true,
+															ForceNew:     true,
+															ValidateFunc: validation.StringIsNotEmpty,
+														},
+													},
+												},
+											},
+
+											"adapter_property_override_enabled": {
+												Type:     pluginsdk.TypeBool,
+												Optional: true,
+												ForceNew: true,
+												Default:  false,
+											},
+
+											"qos_policy_override": {
+												Type:     pluginsdk.TypeList,
+												Optional: true,
+												ForceNew: true,
+												MaxItems: 1,
+												Elem: &pluginsdk.Resource{
+													Schema: map[string]*pluginsdk.Schema{
+														"bandwidth_percentage_smb": {
+															Type:         pluginsdk.TypeString,
+															Optional:     true,
+															ForceNew:     true,
+															ValidateFunc: validation.StringIsNotEmpty,
+														},
+
+														"priority_value8021_action_cluster": {
+															Type:         pluginsdk.TypeString,
+															Optional:     true,
+															ForceNew:     true,
+															ValidateFunc: validation.StringIsNotEmpty,
+														},
+
+														"priority_value8021_action_smb": {
+															Type:         pluginsdk.TypeString,
+															Optional:     true,
+															ForceNew:     true,
+															ValidateFunc: validation.StringIsNotEmpty,
+														},
+													},
+												},
+											},
+
+											"qos_policy_override_enabled": {
+												Type:     pluginsdk.TypeBool,
+												Optional: true,
+												ForceNew: true,
+												Default:  false,
+											},
+
+											"virtual_switch_configuration_override": {
+												Type:     pluginsdk.TypeList,
+												Optional: true,
+												ForceNew: true,
+												MaxItems: 1,
+												Elem: &pluginsdk.Resource{
+													Schema: map[string]*pluginsdk.Schema{
+														"enable_iov": {
+															Type:         pluginsdk.TypeString,
+															Optional:     true,
+															ForceNew:     true,
+															ValidateFunc: validation.StringIsNotEmpty,
+														},
+
+														"load_balancing_algorithm": {
+															Type:         pluginsdk.TypeString,
+															Optional:     true,
+															ForceNew:     true,
+															ValidateFunc: validation.StringIsNotEmpty,
+														},
+													},
+												},
+											},
+
+											"virtual_switch_configuration_override_enabled": {
```

Comment:
this has the block name in it?
```suggestion
											"enabled": {
```

Reply:
The toggle is at the same level to the `virtual_switch_configuration_override` block, and controls whether the override is enabled.